Enable AWS Certificate Manager Support

sevenmachines opened the pull request:
This change adds in ACM support for bootstrap-cfn. This is usable
as an option when projects have access to the admin range of email
addresses on the domain or super-domain. It also includes docs and
a cloudformation stack for setting up an SES->S3->Lambda email
forwarding.

Commit:
Enable AWS Certificate Manager Support

This change adds in ACM support for bootstrap-cfn. This is usable
as an option when projects have access to the admin range of email
addresses on the domain or super-domain. It also includes docs and
a cloudformation stack for setting up an SES->S3->Lambda email
forwarding.

Commit:
pep8 fix to relace lambda function

Commit:
Update ACM resource tagging and documentation

mattpep commented on docs/ses_acm_validation.md:
```diff
@@ -0,0 +1,73 @@
+Using the SESLambdaForwarder cloudformation stack
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There’s a cloudformation template available for doing this, 
```
This is a &rsquo; - it should be &apos; (looks like a weird smart quote issue)

sevenmachines replied:
updated

Commit:
Strip whitespace from input addresses SES lambda forwarder example

Commit:
Pretty format lambda forwarder logging messages

yufangzhang commented on docs/ses_acm_validation.md:
```diff
@@ -0,0 +1,73 @@
+
+http://docs.aws.amazon.com/acm/latest/userguide/gs-acm-validate. html
+
+![alt text](images/ses_acm_ses_sns.png "Certificate manager to simple email service pipeline")
```
think image is in `images/acm_ses_sns.png`...

sevenmachines replied:
think its acm_ses_sns.png :)

yufangzhang commented on docs/ses_acm_validation.md:
```diff
@@ -0,0 +1,73 @@
+sets validation domain to mydomain.com, so the verification emails will not be set to 
+admin@mysite.mysubdomain.superdomain.com, but instead to admin@superdomain.com.
+
+See http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/ aws-resource-certificatemanager-certificate.html
```
a tiny space blocking the url....:p

Commit:
Translate errant rst into markdown in SES readme document

sevenmachines commented:
@yufangzhang made the changes and fixed the docs. The image link should work once its in master

Commit:
Make flake8 ignore parse errors in docs directory

The docs directory is currently parsed by flake8 causing build
errors. This change just makes the docs dir ignored for travis
flake8 tests.
